<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModParamQuery</class>
 <widget class="QDialog" name="CModParamQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询修改记录</string>
  </property>
  <widget class="QLabel" name="label_lineno">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="CKeyButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="CKeyTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="CKeyButton" name="pushButton_query">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>查询日期</string>
   </property>
  </widget>
  <widget class="CKeyDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKeyButton</class>
   <extends>QPushButton</extends>
   <header>UiCommon/ckeybutton.h</header>
  </customwidget>
  <customwidget>
   <class>CKeyTableWidget</class>
   <extends>QTableWidget</extends>
   <header>UiCommon/ckeytablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CKeyDateEdit</class>
   <extends>QDateEdit</extends>
   <header>UiCommon/ckeydateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_query</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
